client/src/components/main/home/panels/components/CardsPanel.js

```diff
@@ -177,40 +177,43 @@ export default class CardsPanel extends React.Component {
             type="actions-container-background"
             className={styles.actionsContainerBackground} />
           {
-            actions.map((action, index, array) => {
-              return (
-                <Row
-                  type="flex"
-                  justify="start"
-                  align="middle"
-                  key={index}
-                  className={styles.actionButton}
-                  onClick={e => this.onActionClicked(e, action, child)}
-                  style={{
-                    flex: 1.0 / array.length,
-                    minHeight: ACTION_MIN_HEIGHT
-                  }}>
-                  <Row type="flex" align="middle">
-                    {
-                      action.icon
-                        ? <Icon style={action.style} type={getIconType(action)} />
-                        : undefined
-                    }
-                    {
-                      action.overlay
-                        ? (
-                          <Popover
-                            onVisibleChange={onVisibleChange}
-                            content={action.overlay}>
-                            <span style={action.style}>{action.title}</span>
-                          </Popover>
-                        )
-                        : <span style={action.style}>{action.title}</span>
-                    }
+            actions
+              .filter((action) => (action.title !== 'PAUSE' && action.icon !== 'fork') ||
```

Comment:
We shouldn't rely on **icon** and **title** while rendering run actions (as they may change); instead we should consider run's **platform** when we create **actions** array and _do not add_ **pause** action for windows-based runs

Reply:
Done!